linux_os/guide/system/software/integrity/crypto/ssh_client_use_strong_rng/rule.yml

```diff
@@ -0,0 +1,44 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'SSH client uses strong entropy to seed'
+
+description: |-
+    To set up SSH client to use entropy from a high-quality source, make sure
+    that appropriate shell environment variables are configured. The
+    <tt>SSH_USE_STRONG_RNG</tt> environment variable determines how many bytes
+    of entropy to use. Make sure that the file
+    <tt>/etc/profile.d/cc-ssh-strong-rng.sh</tt> contains line
+    <pre>export SSH_USE_STRONG_RNG=32</pre>
+    and that the file <tt>/etc/profile.d/cc-ssh-strong-rng.csh</tt> contains line
```

Comment:
there are 2 configuration changes here which means there needs to be separate rules for them.